deploy_and_run_tests.sh

```diff
@@ -220,12 +220,12 @@ fi
 pip install --upgrade .[int_test]
 
 color_print "Running integration tests against ${full_image_name}" "${GREEN}"
-python gcp_variant_transforms/testing/integration/run_vcf_to_bq_tests.py \
-    --project "${project}" \
-    --staging_location "gs://${gs_dir}/staging" \
-    --temp_location "gs://${gs_dir}/temp" \
-    --logging_location "gs://${gs_dir}/temp/logs" \
-    --image "${full_image_name}" ${TEST_ARGUMENTS}
+#python gcp_variant_transforms/testing/integration/run_vcf_to_bq_tests.py \
+#    --project "${project}" \
+#    --staging_location "gs://${gs_dir}/staging" \
+#    --temp_location "gs://${gs_dir}/temp" \
+#    --logging_location "gs://${gs_dir}/temp/logs" \
+#    --image "${full_image_name}" ${TEST_ARGUMENTS}
```

Comment:
Why not just remove them if they're not being used?

Reply:
The latest commit is still in progress and it is not ready for the next round of review yet. I temporarily comment this out simply for faster testing speed. Sorry for the inconvenience. :)

Reply:
No worries, totally understandable :)